airflow/www/security.py

```diff
@@ -21,7 +21,7 @@
 from flask import g
 from flask_appbuilder.security.sqla import models as sqla_models
 from flask_appbuilder.security.sqla.manager import SecurityManager
-from sqlalchemy import or_, and_
+from sqlalchemy import and_, or_
```

Comment:
Looks like unrelated change?

Reply:
That was related to an issue that were isor error on ci/cd and i  did rebase the git and force push it

Reply:
I will close this PR and create another on to fix this.